orchestrator/api/widgets/documents.py

```
@@ -79,66 +86,107 @@ async def search_documents(
+
+    filters = build_retrieval_filters(workspace_id=auth.workspace_id, team=auth.team)
+
+    try:
+        result = await RAGService().retrieve(
+            query=body.query,
+            max_chunks=body.limit,
+            workspace_id=filters.workspace_id,
+            team=filters.team,
+        )
+        chunks = result.chunks if result else []
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)